integration-tests/src/test/scala/za/co/absa/spline/ElasticSearchSpec.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 ABSA Group Limited
```

Comment:
minor: the date of copyright notice, once set, should not be changed

Reply:
fixed